source/_components/sensor.hp_ilo.markdown

```diff
@@ -2,7 +2,7 @@
 layout: page
 title: "HP ILO"
 description: "How to integrate HP ILO (Integrated Lights-Out) sensors within Home Assistant."
-date: 2016-08-15 19:00
+date: 2017-05-10 19:00
```

Comment:
Leave the date as it was since the creation date of this document hasn't changed.